src/Command/MeilisearchClearCommand.php

```diff
@@ -54,6 +53,6 @@ protected function execute(InputInterface $input, OutputInterface $output): int
 
         $output->writeln('<info>Done!</info>');
 
-        return 0;
+        return \Symfony\Component\Console\Command\Command::SUCCESS;
```

Comment:
Not sure if changes to constants add any value

Reply:
Its automated via rector, of course it is to be challenged
To me i am neither +1 nor -1

Reply:
then it breaks CS, as the class should be imported not referenced via FQCN

Reply:
I did only run rector, not any cs fixer tool for now
Of course before i remove the WIP i will check this cs 👍🏻